components/providers/provider-hero-image.tsx

```
@@ -0,0 +1,256 @@
+  const [imageError, setImageError] = useState(false);
+  
+  // Reset image error on new preview URL
+  React.useEffect(() => {
+    if (previewUrl) {
+      setImageError(false);
+    }
+  }, [previewUrl]);
+
+  // Handle file selection
+  const handleFileSelect = useCallback((e: React.ChangeEvent<HTMLInputElement>) => {
+    if (e.target.files && e.target.files[0]) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)